index.js

```diff
@@ -0,0 +1,59 @@
+const core = require('@actions/core');
+// const github = require('@actions/github');
+
+const { workspaces } = require('./package.json');
+const env = 'staging';
+
+try {
+    // `files` input defined in action metadata file
+    const input = core.getInput('files');
+    // We're receiving this stringified, parse it in order to use it as an array
+    const changedFiles = Array.isArray(input) ? input : JSON.parse(input);
+    console.log(`changed files: ${changedFiles}`);
+    const inWorkspace = [];
+    changedFiles.forEach((file) => {
+        let integration;
+        if (file.includes('/')) {
+            integration = file.split('/')[0];
+        } else if (file.includes('\\')) {
+            /* Should support Backslash as well as forward ?*/
```

Comment:
it probably should if this was wide and generic but if it works well like this I think its ok to leave it "hardcoded" because it will always work under the eco system of github
and if not - we can change it